### PR TITLE
Fix no audio on quick join, other small fixes

### PR DESCRIPTION
**Pull request by hensmi-amazon**
**Issue #:** None

**Description of changes:** See title. Self explanatory. I also removed some deprecated constraints and pointless log lines: https://groups.google.com/g/discuss-webrtc/c/85e-f_siCws

**Testing:** Audio now works on quick join.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
Just demo feature.

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
y

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
n

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
n

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 1]
```diff
@@ -707,6 +707,7 @@ export class DemoMeetingApp
           }
           /* @ts-ignore */
           if ((e as SubmitEvent).submitter.id == 'quick-join') {
+            await this.openAudioInputFromSelectionAndPreview();
```
Comment (devalevenkatesh) [inline demos/browser/app/meetingV2/meetingV2.ts:710]: Irrelevant, but can you also make the abort on fatal error as the default option as you have this PR :)

Comment (hensmi-amazon) [reply demos/browser/app/meetingV2/meetingV2.ts:710]: sure!

**Comment by hensmi-amazon** [comment]
I also removed some deprecated constraints and pointless log lines: https://groups.google.com/g/discuss-webrtc/c/85e-f_siCws

File: src/task/CreatePeerConnectionTask.ts [hunk 2]
```diff
@@ -83,17 +83,12 @@ export default class CreatePeerConnectionTask extends BaseTask implements Remova
     configuration.bundlePolicy = this.context.browserBehavior.requiresBundlePolicy();
     // @ts-ignore
     configuration.sdpSemantics = 'unified-plan';
-    // @ts-ignore
-    this.logger.info(`SDP semantics are ${configuration.sdpSemantics}`);
-    const connectionConstraints = {
```
Comment (devalevenkatesh) [inline src/task/CreatePeerConnectionTask.ts:88]: Would it be good to document these constraint removal in changelog? It currently just says deprecated constraints. Also the same link from description can also be good addition to changelog for this I think.

**Review by devalevenkatesh — APPROVED** [review]

**Review by ltrung — APPROVED** [review]

**Pull request merged**